tests/test_setup.py

```diff
@@ -1,124 +1,131 @@
-import sqlite3
-
-import tests.util as util
-
-from indexd.index.drivers.alchemy import SQLAlchemyIndexDriver
-from indexd.alias.drivers.alchemy import SQLAlchemyAliasDriver
-
-
-OLD_SQLITE = sqlite3.sqlite_version_info < (3, 7, 16)
-
-INDEX_HOST = 'index.sq3'
-ALIAS_HOST = 'alias.sq3'
-
+# column name, data type, nullable, default value, primary key
 INDEX_TABLES = {
```

Comment:
Should `index_record_url_metadata_jsonb` be in here, and can we remove `base_version`?

Also, I guess we didn't check for the metadata tables in the past...

Reply:
I didn't even notice that ace was also missing. :c